### DISCUSSION TITLE
Base unit for indent

### DISCUSSION THREAD

**Discussion by ladislasdellinger** [dolanmiu/docx#2161] [Q&A] [answered]
Hello,

I have a small question about something not explained in the documentation.

What is the default unit for the `indent` of a paragraph ?

Thank's in advance for your answer 🙂

**Comment by cwdaff** [comment, marked as answer]
Assuming no tab stop, I expect it should be the default applied by Word, so 0.5 inches aka 720 TWIPs. If I understood the question correctly.

**Reply by dolanmiu** [reply]
Yes I believe so

Read more about it here:

https://en.wikipedia.org/wiki/Twip

**Reply by ladislasdellinger** [reply]
Thank's for your answer, that's what I needed :)
